src/ValueRepresentation.js

```diff
@@ -670,41 +764,85 @@ class PersonName extends EncodedStringRepresentation {
     constructor() {
         super("PN");
         this.maxLength = null;
-        this.padByte = 0x20;
+        this.padByte = PADDING_SPACE;
+    }
+
+    static checkComponentLengths(components) {
```

Comment:
I see value in not writing non-conformant JSON, but reading it should be fully supported - can you test reading non-conformant data and then indicate here that this function only prevents one from creating it?

Reply:
As it stands it's only checked on write. I'll add a comment about it.